src/stories/Button.stories.js

```diff
@@ -1,28 +1,90 @@
 import CircleButton from '../components/button/CircleButton.vue'
+import TextButton from '../components/button/TextButton.vue'
 
 // This is required for each story
 export default { title: 'Button' }
 
 // Customize components here. For instance, here's my-button component with a text of "with text"
 export const basicButton = () => ({
   components: { BasicButton },
-  template: '<basic-button />'
+  template: `
+    <div>
+      <basic-button>Default</basic-button>
+      <basic-button type="primary">Primary</basic-button>
+      <basic-button type="success">Success</basic-button>
+      <basic-button type="info">Info</basic-button>
+      <basic-button type="warning">Warning</basic-button>
+      <basic-button type="danger">Danger</basic-button>
+    </div>  
+  `
+})
+
+export const disabledButton = () => ({
+  components: { DisabledButton },
+  template: `
+  <div>
+    <disabled-button>Default</disabled-button>
+    <disabled-button type="primary">Primary</disabled-button>
+    <disabled-button type="success">Success</disabled-button>
+    <disabled-button type="info">Info</disabled-button>
+    <disabled-button type="warning">Warning</disabled-button>
+    <disabled-button type="danger">Danger</disabled-button>
+  </div>  
+`
 })
 
 export const plainButton = () => ({
   components: { PlainButton },
-  template: '<plain-button />'
+  template: `
+  <div>
+    <plain-button>Default</plain-button>
+    <plain-button type="primary">Primary</plain-button>
+    <plain-button type="success">Success</plain-button>
+    <plain-button type="info">Info</plain-button>
+    <plain-button type="warning">Warning</plain-button>
+    <plain-button type="danger">Danger</plain-button>
+  </div>  
+`
 })
 
 export const roundButton = () => ({
   components: { RoundButton },
-  template: '<round-button />'
+  template: `
+  <div>
+    <round-button>Default</round-button>
+    <round-button type="primary">Primary</round-button>
+    <round-button type="success">Success</round-button>
+    <round-button type="info">Info</round-button>
+    <round-button type="warning">Warning</round-button>
+    <round-button type="danger">Danger</round-button>
+  </div>  
+`
 })
 
 export const circleButton = () => ({
   components: { CircleButton },
-  template: '<circle-button />'
+  template: `
+    <div>
+      <circle-button icon="el-icon-search"></circle-button>
+      <circle-button icon="el-icon-edit" type="primary"></circle-button>
+      <circle-button icon="el-icon-check" type="success"></circle-button>
+      <circle-button icon="el-icon-message" type="info"></circle-button>
+      <circle-button icon="el-icon-star-off" type="warning"></circle-button>
+      <circle-button icon="el-icon-delete" type="danger"></circle-button>
+    </div>  
+  `
+})
+
+export const textButton = () => ({
+  components: { TextButton },
+  template: `
+  <div>
+    <text-button type="text"></text-button>
+    <text-button type="text" disabled></text-button>
```

Comment:
Without declaring the property in the component, any extra props have no effect. So you have to add
a prop called `disabled` of type `Boolean` to the component. Ideally, we should inherit the properties of the root element so that we don't have to redeclare the props on our custom components.

Reply:
@andrewtpham I've left some comments on slack concerning this.